Amasty_GdprCookie/web/template/cookie-table.html

```diff
@@ -0,0 +1,97 @@
+<div
+    css="modalClass"
+    hasFocus="focused"
+>
+    <p
+        class="amgdprcookie-description"
+        if="$data.description"
+        data-bind="text: $data.description"
+    ></p>
+    <table class="table">
```

Comment:
![image](https://user-images.githubusercontent.com/49198312/130200868-fc123536-8b95-4e4a-a9df-b7712c267494.png)
this table isn't really working well on mobile when it's inside a modal. let's get rid of the modal and put tables inside dropdowns like it was done here https://stage23-colibri-ee.snowdog.dev/cookie-settings . please move  table to `Amasty_GdprCookie/web/template/widget/settings.html`. you can use `<!-- ko foreach: $data.cookies -->` to get table data.